web/packages/api/src/status.ts

```diff
@@ -65,15 +65,17 @@ export enum AlarmReason {
     AccountBalanceInsufficient = "AccountBalanceInsufficient",
     ToEthereumNoTransfer = "ToEthereumNoTransfer",
     ToPolkadotNoTransfer = "ToPolkadotNoTransfer",
+    ToEthereumChannelAttacked = "ToEthereumChannelAttacked",
+    ToPolkadotChannelAttacked = "ToPolkadotChannelAttacked"
 }
 
 export type Sovereign = { name: string; account: string; balance: bigint; type: SourceType }
 
 export const BlockLatencyThreshold = {
-    // Syncing beefy finality update every 4 hours(2400 blocks) so we set 3000 blocks at most.
-    ToEthereum: 3000,
-    // Syncing beacon finality update every 6.4 minutes(32 blocks) so we set 128 blocks (4 epochs) at most.
-    ToPolkadot: 128,
+    // Syncing beefy finality update every 4 hours(1200 ethereum blocks)
+    ToEthereum: 1200,
+    // Syncing beacon finality update every 1 hours(600 substrate blocks)
+    ToPolkadot: 600,
```

Comment:
We haven't changed beacon syncing to an hour, right?

Reply:
Right, mistake that with westend beacon [config interval](https://bridgehub-westend.subscan.io/event?page=1&time_dimension=date&module=ethereumbeaconclient), thanks for catching.